plerkle/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "plerkle"
 description = "Geyser plugin with dynamic config reloading, message bus agnostic abstractions and a whole lot of fun."
-version = "1.6.0"
+version = "1.7.0"
```

Comment:
Why not
```suggestion
version = "1.6.1"
```
?

Reply:
@StanChe The same thing with `plerkle` crate